recipe/recipe.yaml

```
@@ -0,0 +1,51 @@
+schema_version: 1
+
+context:
+  name: zfit-physics
+  version: 0.9.0
+  python_min: 3.10
```

```suggestion
  version: "0.9.0"
  python_min: "3.10"
```

It is preferred that these are quoted given YAML and numbers.